stderr of test 'sequences-defaults.Bug-6744` in directory 'sql/test/BugTracker-2019` itself:


# 11:21:15 >  
# 11:21:15 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34651" "--set" "mapi_usock=/var/tmp/mtest-1370/.s.monetdb.34651" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-Apr2019/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2019" "--set" "embedded_r=yes" "--set" "embedded_py=2" "--set" "embedded_c=true"
# 11:21:15 >  

# builtin opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Apr2019/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 34651
# cmdline opt 	mapi_usock = /var/tmp/mtest-1370/.s.monetdb.34651
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Apr2019/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2019
# cmdline opt 	embedded_r = yes
# cmdline opt 	embedded_py = 2
# cmdline opt 	embedded_c = true
#main thread:!ERROR:MALException:client.quit:Server stopped

# 11:21:16 >  
# 11:21:16 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-1370" "--port=34651"
# 11:21:16 >  

MAPI  = (monetdb) /var/tmp/mtest-30932/.s.monetdb.31943
QUERY = create sequence seq6 MAXVALUE -3; --error, the default values don't match
ERROR = !CREATE SEQUENCE: start value is higher than the maximum (1 > -3)
CODE  = 42000

# 11:21:16 >  
# 11:21:16 >  "Done."
# 11:21:16 >  

